modules/module-extras.php

```diff
@@ -69,3 +69,27 @@ function jetpack_widgets_add_suffix( $widget_name ) {
 	return sprintf( __( '%s (Jetpack)', 'jetpack' ), $widget_name );
 }
 add_filter( 'jetpack_widget_name', 'jetpack_widgets_add_suffix' );
+
+add_action( 'blog_privacy_selector', 'priv_notice_privacy_selector' );
+
+/**
+ * Echos notice directing site owners to Jetpack's Private Site feature.
+ */
+function priv_notice_privacy_selector() {
+	?>
+	<p>
+		<?php
+		wp_kses(
+			sprintf(
+				/* translators: URL to the Jetpack dashboard. */
+				__( 'You can also make your site completely private by allowing only registered users to see your site.  Activate Jetpack\'s Private Site feature <a href="%s">here</a>.', 'jetpack' ),
+				esc_url( admin_url( 'admin.php?page=jetpack' ) . '#/security?term=private' )
+			),
+			array( 'a' => array( 'href' => true ) )
```

Comment:
This could be broken down to remove `wp_kses` and leave the `<a>` outside the translatable string, like this:

```php
<?php
    printf(
        '%s <a href="%s#/security?term=private">%s</a>',
        esc_html__( 'You can also make your site completely private by allowing only registered users to see your site.', 'jetpack' ),
        esc_url( admin_url( 'admin.php?page=jetpack' ),
        esc_html__( "Go to Private Sites", 'jetpack' )
    );
```

Reply:
That's clever. This was my addition to bring things up to PHPCS/WP Coding Standards.

Reply:
Done